Assets/StreamingAssets/Overlay/overlay_functions.lua

```diff
@@ -24,17 +24,18 @@ function roomNumberValueAt ( tile )
 end
 
 -- Returns a magic value if furniture in tile is a power gen
-function powerValueAt(tile)
+function powerValueAt (tile)
+	zero = 153 -- This is middle between 0 and 256
 	if (tile == nil or tile.furniture == nil or tile.furniture.PowerConnection == nill) then
-		return 0
+		return zero
 	end
 	if (tile.furniture.PowerConnection.IsPowerConsumer) then
-		return tile.furniture.PowerConnection.InputRate * -1
+		zero = zero - tile.furniture.PowerConnection.InputRate
```

Comment:
Maybe you want a multiplier for `tile.furniture.PowerConnection.InputRate` (and similarly below)? It's working fine but the color range just isn't that useful at the moment (powerValue only seems to vary by a range of about +/- 1 out of ~150). Or am I testing wrong?